html_challenge1.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Reminder, just like code, you should properly indent if there is an html tag inside of another. This helps you to debug later and it makes it much easier to read.